agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/processors/ExporterWithLogProcessorTest.java

```diff
@@ -184,7 +184,7 @@ void multiRuleToAttributesTest() {
     LogRecordExporter logExporter = new ExporterWithLogProcessor(config, mockExporter);
     TestLogRecordData mockLogA =
         TestLogRecordData.builder()
-            .setBody("yyyPassword=123 aba Pass=555 xyx Pass=777 zzz")
+            .setBody("yyyPassword=123 aba Pass=555 xyx")
```

Comment:
it seems nice to keep the multi-match example here so we can see/confirm the result?